CLI/3. Tutorials/1. 自動完成運算及刪除容器.md

```diff
@@ -1 +1,76 @@
+tags: HowTo, CLI, CCS, EN
+GA: UA-155999456-1
+---
 
+{%hackmd @docsharedstyle/default %}
+
+# HowTo：Use TWCC CLI to automatically complete computing and delete containers
+
+:::success
+<i class="fa fa-star" aria-hidden="true"></i> **Case scenario:Is there a set of workflows that can automatically create containers, perform model training, and delete containers?**
+*Do you have the same problem? Let us take you hand in hand to assemble the TWCC cloud service architecture to easily solve your problems!*
+:::
+This article will help users understand how to use  [TWCC CLI](https://man.twcc.ai/@twccdocs/twcc-cli-v05) **to automate the following tasks:**
+
+```
+1. Execute computing program
+2. Delete container
+```
+In portal website or using TWCC CLI to create [Interactive Container](https://www.twcc.ai/doc?page=container), You can assign manual tasks to script to complete automatically, delete containers, and reduce the cost of continuous opening of containers. The computing program and data will be kept in the default storage space of the container - [Hyper File System (HFS)](https://www.twcc.ai/doc?page=hfs), and no need to worry that the data will disappear when the container is deleted.
+
+
+### Step 1. Upload the computing program to the container
+
+Refer to [<ins>Using SFTP + Filezilla to transfer file</ins>](https://man.twcc.ai/@twccdocs/rypYCr_TN?type=view#%E4%BD%BF%E7%94%A8-SFTP--Filezilla-%E5%82%B3%E8%BC%B8%E6%AA%94%E6%A1%88) to upload the training program to the container's default storage space-Hyper File System (HFS) storage space.
+
+
+### Step 2. Deployment of TWCC CLI
+
+Refer to [<ins>1. Deployment of Operating System</ins>](https://man.twcc.ai/@twccdocs/twcc-cli-v05#1-%E9%83%A8%E7%BD%B2%E6%93%8D%E4%BD%9C%E7%92%B0%E5%A2%83) to deploy Interactive Container using TWCC CLI.
+
+
+### Step 3. Viewing Interactive Container's ID
+- Enter the following command to view container's ID that is currently using:
+```
+$ twccli ls ccs
+```
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_9dbab117e8ee86d346497b2296382ed2.png)
+
+
+### Step 4. Writing automation script
+
+- Enter the following command to edit the automation script:
+```
+$ vi auto.sh
+```
+- Enter `i` to enter the editing mode, copy and paste the following example script into `auto.sh`. The content can be modified according to your computing program.
+
+```bash
+TWCC_CLI_CMD=/home/<USERNAME>/.local/bin/twccli
+#<USERNAME>：Supercomputer Account
+
+echo "1. Execute computing"
+#Enter the executable file of the computing program
+
+echo "2. Delete Interactive Container"
+$TWCC_CLI_CMD rm ccs -f -s <CCS_ID>
+#<CCS_ID>：Step 3' container ID
+
+```
+- Press <kbd>Esc</kbd> key and enter `:wq!` to save.
+
+### Step 5. Execute automated script
+
+Enter the following command to letting the script to handle the program computing and delete container job!
+
+```
+$ bash auto.sh
+```
+
+### Step 6. Confirm that the container has been automatically deleted
```

Comment:
[SUG]

Confirm that the container has been automatically deleted > 

Confirm the container has been automatically deleted